sevntu-checks/src/test/java/com/github/sevntu/checkstyle/internal/AllChecksTest.java

```diff
@@ -236,14 +236,14 @@ private static void grabAllTests(Map<String, List<String>> allTests, File file)
             }
 
             final int slash = path.lastIndexOf(File.separatorChar);
-            final String packge = path.substring(0, slash);
+            final String pkg = path.substring(0, slash);
```

Comment:
Using `packge` just looks like a spelling error, whereas `pkg` looks intentional.

However, I can understand if you're attached to this spelling...